### DISCUSSION TITLE
Move the position of the status bar to the top

### DISCUSSION THREAD

**Discussion by Eloquencere** [dj95/zjstatus#88] [General]
Is there a way to move the position of a particular status bar config.

**Comment by Eloquencere** [comment]
I also have an additional query, is it possible to make visible certain information only when in a certain mode? Say, I want the session name visible only when in the session mode and hide all the tabs

**Reply by dj95** [reply]
Unfortunately, that is not possible and I don't really see, how I could implement this. IMO the configuration language is too limited, to introduce such a feature.

But I really like the idea! Maybe I'll get some idea how to implement it. Thanks for sharing!

**Comment by dj95** [comment]
Please take a look at the zellij layout documentation and the closed issues in zjstatus.

There was an issue with a similar question and the answer :)

**Reply by Eloquencere** [reply]
Yeah, sorry about that, I managed to figure out how to do it.

**Reply by dj95** [reply]
No worries! :) I just like to direct people to hints because it helps to find a solution for other problems more easily in the future instead pasting the plain solution.

**Reply by nikaro** [reply]
For internet searchers here it is: https://github.com/dj95/zjstatus/issues/15#issuecomment-1766776779
